src/Options.purs

```diff
@@ -193,6 +203,42 @@ mkStringArray :: String -> NullOrUndefined [String] -> [String]
 mkStringArray key opt = concat $ mkString key <$> (NullOrUndefined <<< Just)
                                               <$> (fromMaybe [] $ runNullOrUndefined opt)
 
+mkPathArray :: String -> NullOrUndefined [String] -> [String]
+mkPathArray key opt = concat $ mkString key <$> (NullOrUndefined <<< Just)
+                                            <$> (fromMaybe [] (runNullOrUndefined opt) >>= expandGlob)
+
+mkDocgen :: String -> NullOrUndefined Foreign -> [String]
```

Comment:
I'm not necessarily sure the implementation of this option is in the spirit of things, so if you have a better idea for how it might be done, please say!